quinn-proto/src/connection/mod.rs

```diff
@@ -362,6 +369,66 @@ impl Connection {
         this
     }
 
+    /// Set up qlog for this connection.
+    #[cfg(feature = "qlog")]
+    pub fn set_qlog(
+        &mut self,
+        writer: Box<dyn std::io::Write + Send + Sync>,
+        title: Option<String>,
+        description: Option<String>,
+        now: Instant,
+    ) {
+        let vp = if self.side.is_server() {
```

Comment:
Nit: let's import all of these types. Call the `vp` binding `ty` instead to match the upstream name?